### ISSUE TITLE
Cant switch to output binding

### ISSUE THREAD

**Issue by SevrenMusic** [nhielost/obs-midi-mg#70]
Im trying to create an output binding in obs-midi-mg but cannot get the switch to output binding to light up. Can you please tell me how to create an output binding in obs-midi-mg to trigger songs in Cantabile in sync with videos in OBS?
![Screenshot (36)](https://github.com/nhielost/obs-midi-mg/assets/135540625/64b05d71-53ed-478c-bb28-7f69bae24bf1)

**Comment by SevrenMusic** [comment]
Thank you. Your plugin works great. 
I'm using Cantabile to sing and play guitar along with recorded backing tracks. I created bindings in Cantabile to start, stop, and change scenes in OBS. Those all work but the music video playback in OBS is way out of sync with the music in Cantabile and the video has rendering lag. OBS is also using 80% CPU power, so it seems somethings is not right. I posted on OBS site but no answer. I've tried everything I can think of. If you have any ideas on how to correct this, it would be greatly appreciated. Thats why I thought of using OBS to trigger Cantabile since the videos play perfectly in OBS alone.
Thanks
